crates/biome_analyze/CONTRIBUTING.md

```diff
@@ -62,13 +62,13 @@ Let's say we want to create a new rule called `myRuleName`, which uses the seman
    ```
    The script will create a new **lint** rule for the _JavaScript_ language, inside the `biome_js_analyze`
 
-1. The `Ast` query type allows you to query the AST of a program.
-1. The `State` type doesn't have to be used, so it can be considered optional. However, it has to be defined as `type State = ()`.
-1. Implement the `run` function:
```

Comment:
This was intentional, can you rever it please? If you list all numbers to `1.` they will **rendered** automatically in order. This means you just need to add another `1` in case you're adding a new step in *middle*.

It's really convenient.

Reply:
oh, I didn't know that. I restore the change!